deploy/webhook.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-webhook-service
+  namespace: open-cluster-management
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 9443
+  selector:
+    control-plane: controller-manager
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: webhook-serving-cert
```

Comment:
Can this name be made unique since there are other controllers running the same namespace? We don't want conflicts.

Reply:
How about `webhook-propagator-serving-cert`?

Reply:
If `governance-policy-propagator-` prefixes fit, then it'd be good to be consistent. Otherwise, prefixing all with `propagator-` is good.

Reply:
`governance-policy-propagator-` too long, let's go with  `propagator-`